Tools/GNUMake/comps/nvcc.mak

```diff
@@ -112,6 +112,12 @@ ifeq ($(CUDA_VERBOSE),TRUE)
   NVCC_FLAGS += --ptxas-options=-v
 endif
 
+ifeq ($(USE_CUPTI),TRUE)
+  INCLUDE_LOCATIONS += $(CUDAPATH)/extras/CUPTI/include
+  LIBRARY_LOCATIONS += ${CUDAPATH}/extras/CUPTI/lib64 
+  LIBRARIES += -Wl,-rpath,${CUDAPATH}/extras/CUPTI/lib64 -lcupti
```

Comment:
This compiles on Summit but not Cori, is there a system-independent alternative to `$(CUDAPATH)` ?

Reply:
In the sites/Make.nersc there is a SYSTEM_CUDA_PATH and COMPILE_CUDA_PATH set, is that the variable you're looking for?
https://github.com/AMReX-Codes/amrex/blob/83e214b59c54649ea70612e040b2d514c43e961f/Tools/GNUMake/sites/Make.nersc#L64

Reply:
Yes, that's what I needed!  Thanks.